c7n/resources/account.py

```diff
@@ -1453,3 +1453,46 @@ def get_target(self, resource):
                 return []
             resource[self.annotation][kmskey] = self.data[kmskey]
         return resource[self.annotation]
+
+
+@actions.register('set-glue-catalog-encryption')
+class GlueDataCatalogEncryption(BaseAction):
+    """Modifies glue data catalog encryption based on specified
+    parameter
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: data-catalog-encryption
+                resource: account
+                filters:
+                  - type: glue-security-config
+                    CatalogEncryptionMode: DISABLED
+                actions:
+                  - type: set-glue-catalog-encryption
+                    attributes:
+                        EncryptionAtRest:
+                            CatalogEncryption: SSE-KMS
+                            SseAwsKmsKeyId: alias/aws/glue
```

Comment:
Don' think using an alias directly is working in this case! There's no error that pops up but I think it needs a valid kms key arn! I'm investigating!

Reply:
Looks fine, works with alias! I have using a key that did not exist, which confused me. In the console, I saw an error  message asking for a valid key arn

Reply:
![image](https://user-images.githubusercontent.com/22061271/78390012-cdc32180-75b1-11ea-87e3-eaa64208aefc.png)
Tested by running a policy locally